.evergreen/run-orchestration.sh

```diff
@@ -98,10 +114,11 @@ if ! curl --silent --show-error --data @"$ORCHESTRATION_FILE" "$ORCHESTRATION_UR
   exit 1
 fi
 cat tmp.json
-URI=$(python -c 'import json; j=json.load(open("tmp.json")); print(j["mongodb_auth_uri" if "mongodb_auth_uri" in j else "mongodb_uri"])' | tr -d '\r')
+
+URI=$(${PYTHON:-python} -c 'import json; j=json.load(open("tmp.json")); print(j["mongodb_auth_uri" if "mongodb_auth_uri" in j else "mongodb_uri"])' | tr -d '\r')
 echo 'MONGODB_URI: "'$URI'"' > mo-expansion.yml
 echo $URI > $DRIVERS_TOOLS/uri.txt
-echo "Cluster URI: $URI"
+printf "\nCluster URI: $URI\n"
```

Comment:
```suggestion
printf "\nCluster URI: %s\n" "$URI"
```
Recommend [avoiding substitutions in the format string](http://mywiki.wooledge.org/BashPitfalls#printf_.22.24foo.22).